src/classes/STG_InstallScript.cls

```diff
@@ -332,4 +337,19 @@ global without sharing class STG_InstallScript implements InstallHandler {
             UTIL_OrgTelemetry.dispatchTelemetryBatchJob();
         }
     }
+    
+    /**************************************************************************************************************************
+    * @description Enables prevent deletion by default on Hierarchy Custom Settings for new installs and when upgrading to 1.95
+    * @return void
+    */
+    global static void enablePreventDeletionHierarchyCustomSettings(Version previousVersion) {
+        if ((previousVersion == NULL) || (previousVersion != null && previousVersion.compareTo(new Version(1,95)) < 0)) {
```

Comment:
Let's make sure we are consistent with "NULL" or "null"

Reply:
Also, quick question, is there a reason behind not using custom setting facade. configSetting()?

Reply:
Hmm looks like that's how we have been using for other methods too like populateAdminRecordTypeSetting, setDefaultContactLanguageFluency @xinnanlisf

Reply:
updated based on feedback.